CityOfDetroit/bloom
# Refactor of Favorites feature

## Issue

- Addresses #78 

## Description

Refactor of the frontend and backend favorites feature. When a user logs in, they can now favorite or unfavorite listings from the listings page.

![image](https://user-images.githubusercontent.com/82653098/147258494-914d79f4-47b5-4365-8255-be37ab004172.png)

## Type of change


## How Can This Be Tested/Reviewed?

View the console log while adding and removing listings while signed in.

You will see the favorites listing array being added to and removed from.  You can also view the user_preference table in the backend to see the information stored. 

## Checklist:
